WordPress/gutenberg
# Icons: Update the border icon

## What?
Updates the border icon to the new one in https://github.com/WordPress/gutenberg/pull/45995/files

## Why?
It looks more like a border!

## How?
Updates the SVG

## Testing Instructions
Open Global Styles
Navigate to a button block
Check the icon for the border panel

### Testing Instructions for Keyboard
I don't know, sorry :(

## Screenshots or screencast 
<img width="280" alt="Screenshot 2022-12-02 at 08 38 52" src="https://user-images.githubusercontent.com/275961/205251803-3d60d789-b62c-4488-90c8-b3c0ed62096d.png">
